_locales/en/messages.json

```diff
@@ -1748,6 +1757,12 @@
     "pidTuningProfileTip": {
         "message": "Up to 3 (2 for some controllers) different profiles can be stored on the flight controller. The profiles include all the settings on this tab. Once in the field, simple stick commands (see online instructions) can be used to switch between the profiles."
     },
+    "pidTuningRateProfile": {
+        "message": "Rateprofile"
+    },
+    "pidTuningRateProfileTip": {
+        "message": "Up to 3 different rateprofiles can be stored on the flight controller. The rateprofiles include the settings for 'RC Rate', 'Rate', 'RC Expo', 'Throttle', and 'TPA'. Switching between rateprofiles is possible in-flight, by setting up a 3 position switch for 'Rate Profile Selection' on the 'Adjustments' tab."
+    },
```

Comment:
There is actually 3 * (number of profiles) rateprofiles, so maybe the text should be "Up to 3 different rateprofiles per profile can be stored.." or similar ?